### DISCUSSION TITLE
I-Format tape question

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#132] [General]
What are the 32 bits ahead of VOL1? 5000 0000 hex in nos287-1.tap as an example.

**Comment by kej715** [comment]
DtCyber implements a slightly modified version of the SIMH tape container
format. See:

        http://simh.trailing-edge.com/docs/simh_magtape.pdf

The main distinction is that DtCyber doesn't pad records to even length
when writing tape images. It will recognize images that have been padded to
even length.

**Comment by KC5VDJ** [comment]
Thanks, that answers several questions.

Hmm.  Unless I skimmed it wrong, it specifies a 32 bit 0 for a tape mark.  The 0x5000 0x0000 appears again before EOF1 on tape 3.  I-format requires a tape mark before EOF1.

Starting to make sense.  Was thinking Abt this a few nights ago reading about scope tape formats, and noting the scope 2 staging codes performing similar function.

**Comment by drboone** [comment]
Block length, data, repeat block length.  File and error marks are represented as reserved length values, and are not repeated.
